src/Blog/View/posts.twig

```diff
@@ -15,8 +22,15 @@
 		<h1>Blog</h1>
 		<ul>
 			{% for post in posts %}
-				<li>
-					<a href="/blog/{{ post.id }}">{{ post.title }}</a>
+				<li class="u-margin-medium">
+					<a class="post-box__img" href="/blog/{{ post.id }}"> <img src="{{post.url}}"/> </a>
+					<div class="post-box__description">
+					<a class="post-description__header" href="/blog/{{ post.id }}"> <h2 class="post-description__title"> {{post.title}}</h2> <span class="post-descritpion__date"> {{post.date}} </span> </a>
+				
+					<p class="post-description__preamble"> {{post.preamble}} </p>
+		 		    <a class="read-more-button" href="/blog/{{ post.id }}"> <h3> Read more </h3> <img class="move-forward-icon" src="/img/arrow.png"/> </a>
```

Comment:
Add "clearfix" class selector to this element. It should look like this:
```
class="read-more-button clearfix"
```

Reply:
Change `h3` element to `span`. 
Span element as default has got "display: inline", but `h3` has got default "display: block". The different is that, `display: inline` take only content place, but `dislpay: block` take all avaliable place. acording to that, we need here span element.

Reply:
You can add arrow icon to your repository to public/image/icon-back.svg direction. You will find this icon in trello inside the description for this task.

Reply:
done